components/canary/chart/templates/ingress.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
Aren't we using istio `Gateway` and `VirtualService` for this now? This file can probably be removed...

Reply:
gone